settings.gradle

```diff
@@ -12,13 +12,6 @@ pluginManagement {
 
 rootProject.name = 'opensearch-data-prepper'
 
-if(startParameter.getProjectProperties().containsKey('release')){
-    include 'release'
-    include 'release:archives'
-    include 'release:archives:linux'
-    include 'release:docker'
-    include 'release:maven'
-}
```

Comment:
With the recent directory change, I think this conditional is not needed anymore. We need `release` projects included to assemble a runnable data prepper because the `data-prepper` shell script is inside `release` folder. 

Anything I may be missing here?

Reply:
I agree here that we shouldn't require this property anymore.

The only possible downsides I would see are: 1) If the release build requires build parameters the build would fail, but that seems to not be the case; 2) Users might have to download a few extra dependencies to build (e.g. Docker), but that should be quite minor.

I think the benefit of a streamlined command outweighs those.

Reply:
Is this not required now because we want to create a runnable distribution even with `./gradlew assembe`?

Reply:
Yes, that's right.